### PR TITLE
qa/rgw_snaps: move default_idle_timeout config under the client

### PR DESCRIPTION
**Pull request by yehudasa**
Fixes: http://tracker.ceph.com/issues/20128

Signed-off-by: Yehuda Sadeh <yehuda@redhat.com>

### CODE REVIEW

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2017-06-01_17:53:43-rados-wip-sage-testing2-distro-basic-smithi/

File: qa/suites/rados/basic/tasks/rgw_snaps.yaml [hunk 1]
```diff
@@ -8,8 +8,8 @@ overrides:
         osd_max_omap_entries_per_request: 10
 tasks:
 - rgw:
-    default_idle_timeout: 3600
-    client.0: null
+    client.0:
+      default_idle_timeout: 3600
```
Comment (cbodley) [inline qa/suites/rados/basic/tasks/rgw_snaps.yaml]: looks like i broke this in https://github.com/ceph/ceph/pull/15184 - `default_idle_timeout` was specific to fastcgi, so it's no longer being parsed and removed from `config`. this change should work, but it's probably better to remove the setting completely

it looks like the same issue is present in other upgrade tests:
> rados/upgrade/jewel-x-singleton/8-workload/rgw-swift.yaml:    default_idle_timeout: 300
upgrade/jewel-x/stress-split/7-final-workload/rgw-swift.yaml:    default_idle_timeout: 300
upgrade/kraken-x/stress-split/7-final-workload/rgw-swift.yaml    default_idle_timeout: 300

mind removing those too?

Comment (yehudasa) [inline qa/suites/rados/basic/tasks/rgw_snaps.yaml]: @cbodley @liewegas I don't mind removing them altogether. I'm not sure we'd see the same issue in those tests, I think the problem we were seeing here was that the field was at the wrong level in the hierarchy.

Comment (cbodley) [inline qa/suites/rados/basic/tasks/rgw_snaps.yaml]: the `default_idle_timeout` field was global to the rgw task (see https://github.com/ceph/ceph/pull/15184/files#diff-995b04809fcabacc3e3ecfaea903a41aL1128). there was also an `idle_timeout` that could be used per-client

those global fields are removed from `config` as they're parsed, resulting in a `config` whose only keys are clients. that's why it's now being mistakenly interpreted as a remote in this test failure. i'd expect the same thing to happen in the upgrade tests

moving the line under client.0 does work to pass the test - it just won't be used anywhere

**Comment by liewegas** [comment]
merging this as is just to fix the breakage!

**Pull request merged**